tests/unit/keyinput/test_keyutils.py

```diff
@@ -452,7 +452,12 @@ def test_with_mappings(self):
         ('<alt+less>',
          keyutils.KeySequence(Qt.Key_Less | Qt.AltModifier)),
 
-        ('<alt+<>', keyutils.KeyParseError),
+        ('<<', keyutils.KeySequence(Qt.Key_Less, Qt.Key_Less)),
+        ('>>', keyutils.KeySequence(Qt.Key_Greater, Qt.Key_Greater)),
+        ('><', keyutils.KeySequence(Qt.Key_Greater, Qt.Key_Less)),
+        ('<alt+<>',
+         keyutils.KeySequence(Qt.Key_Less | Qt.AltModifier)),
```

Comment:
On the german keyboard layout `alt+<` yields `|`
To get `>` requires `shift+<`
`alt+shift+<` does not yield a key input

I'm afraid this needs some more care to be internationally compatible